org.eclipse.scanning.api/src/org/eclipse/scanning/api/event/scan/DeviceRequest.java

```diff
@@ -208,24 +230,29 @@ public DeviceRequest(String name, DeviceType type, DeviceAction action, Object d
 	public void setDevices(Collection<DeviceInformation<?>> devices) {
 		this.devices = devices;
 	}
-	
 
 	@Override
 	public int hashCode() {
 		final int prime = 31;

```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)